<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programmer 2020</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_main">
     <attribute name="title">
      <string>Главная</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>19</y>
        <width>551</width>
        <height>388</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="followProjectLabel">
           <property name="text">
            <string>Текущий проект: Нет проекта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressTime">
           <property name="maximum">
            <number>876000</number>
           </property>
           <property name="value">
            <number>219000</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="format">
            <string>Время</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMoney">
           <property name="text">
            <string>Деньги: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHealth">
           <property name="text">
            <string>Здоровье:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIntellect">
           <property name="text">
            <string>Интеллект:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMood">
           <property name="text">
            <string>Настроение:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelReputation">
           <property name="text">
            <string>Репутация:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Навыки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="skillsList">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="labelAvatar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>330</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/res/images/steve-halama.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_learning">
     <attribute name="title">
      <string>Learning</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>19</y>
        <width>551</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="readBookButton">
         <property name="text">
          <string>Прочитать книгу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="learnCourseButton">
         <property name="text">
          <string>Изучить онлайн курс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getAnswerButton">
         <property name="text">
          <string>Найти ответ на Stackoverflow</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_work">
     <attribute name="title">
      <string>Работа</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>651</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageJobMain">
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>551</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="takeProjectButton">
           <property name="text">
            <string>Выполнить заказ на фрилансе</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findJobButton">
           <property name="text">
            <string>Найти работу</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="pageFindJob">
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>551</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Работа:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="jobList"/>
         </item>
         <item>
          <widget class="QPushButton" name="doJobButton">
           <property name="text">
            <string>Выполнить работу</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_holiday">
     <attribute name="title">
      <string>Отдых</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>19</y>
        <width>551</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Выбор места отдыха:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="restList"/>
       </item>
       <item>
        <widget class="QPushButton" name="chooseRestButton">
         <property name="text">
          <string>Поехали!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
